superset-frontend/src/common/components/index.tsx

```diff
@@ -33,8 +33,10 @@ export {
   DatePicker,
   Dropdown,
   Empty,
+  Input,
   Modal,
   Popover,
+  Select,
```

Comment:
Are these Input/Select elements significantly different from a visual perspective? We should consider restyling them as needed to look like the SIP-34 design system, and eventually replacing the other disparate Input/Select components with these ones for consistency. Probably not a rush, but I just want to make sure we follow up on this stuff when we add new species :D

Reply:
The difference is not that significant. Still, when react-bootstrap input was next to Antd Select, it looked weird.
Before:
![image](https://user-images.githubusercontent.com/15073128/98266566-c40bfd00-1f8a-11eb-91e5-20fd91d929d8.png)
After:
![image](https://user-images.githubusercontent.com/15073128/98266051-1ef12480-1f8a-11eb-979d-6a50cf529e87.png)